docs/guides/custom-functions.md

```diff
@@ -183,7 +183,7 @@ export default (obj) => {
 };
 ```
 
-The following code won't work properly either:
+Require calls will work only in Node.js, therefore it's strongly discouraged to use them unless you don't plan to distribute your ruleset among other users:
```

Comment:
>strongly discouraged to use them unless you don't plan

@P0lip This double negative statement is a tad hard to parse and be 100% sure we understand it right the first time.

How about something like this:

Require calls will work only in Node.js. They won't work in browser based context. Thus, it's highly recommended to not rely on require calls if you plan to distribute your ruleset.

Reply:
I made a suggestion, think I got it.